presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/framework/BenchmarkRunner.java

```diff
@@ -53,20 +57,30 @@ public void start()
         BenchmarkSuite benchmarkSuite = benchmarkSuiteSupplier.get();
         BenchmarkSuiteInfo benchmarkSuiteInfo = benchmarkSuite.getSuiteInfo();
         List<PhaseSpecification> phases = benchmarkSuiteInfo.getPhases();
+        int successfulPhases = 0;
```

Comment:
Remove

Reply:
We need this to figure out the type of the `BenchmarkSuiteEvent`. This is a local variable and tells us whether the suite SUCCEEDED or COMPLETED_WITH_FAILURES when the `continueOnFailure` is `true`. So i think we need this ?